docs/get_cover_gallery.py

```diff
@@ -7,11 +7,70 @@
 style.use("preliz-doc")
 rng = np.random.default_rng(247)
 
+category_colors = {
```

Comment:
Any reason for this particular order of the colors?

Reply:
No reason. I just liked that order. But we can change it